# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1280821252 -7200
#      Tue Aug 03 09:40:52 2010 +0200
# Node ID d782d7b524a655e6dbe772f36bc458fb3088aa6c
# Parent  7fa3c8e43357a5a69840f167bb88626aa03bb826
document the nargout=0 exception in isargout

diff --git a/src/ov-usr-fcn.cc b/src/ov-usr-fcn.cc
--- a/src/ov-usr-fcn.cc
+++ b/src/ov-usr-fcn.cc
@@ -746,21 +746,21 @@ static bool isargout1 (int nargout, cons
     }
   else
     return (k == 1 || k <= nargout) && ! val_in_table (ignored, k);
 }
 
 DEFUN (isargout, args, ,
   "-*- texinfo -*-\n\
 @deftypefn {Built-in Function} {} isargout (@var{k})\n\
-Within a function, given an index @var{k} within the range @code{1:nargout},\n\
+Within a function, given an index @var{k} within the range @code{1:max(nargout,1)},\n\
 return a logical value indicating whether the argument will be assigned on\n\
 output to a variable or cell or struct element. If the result is false,\n\
 the argument will be ignored using the tilde (~) special output argument.\n\
-If @var{k} is outside the range @code{1:nargout}, the function yields false.\n\
+If @var{k} is outside the range @code{1:max(nargout,1)}, the function yields false.\n\
 @var{k} can also be an array, in\n\
 which case the function works element-wise and a logical array is returned.\n\
 At the top level, @code{isargout} returns an error.\n\
 @seealso{nargout, nargin, varargin, varargout}\n\
 @end deftypefn")
 {
   octave_value retval;
 
